doc/docs/modules/ROOT/pages/dwh.adoc

```diff
@@ -0,0 +1,608 @@
+  .load()
+
+// Step (2)
+// Save the `bigqueryDF` as nodes with labels `Person` and `Customer` into Neo4j
+bigqueryDF.write
+  .format("org.neo4j.spark.DataSource")
+  .mode(SaveMode.ErrorIfExists)
+  .option("url", "neo4j://<host>:<password>")
+  .option("labels", ":Answer")
+  .load()
+----
+=====
+[.include-with-python]
+=====
+[source,python]
+----
+# Step (1)
+# Load a table into a Spark DataFrame
+bigqueryDF = (spark.read
+  .format("bigquery")
+  .option("table", "google.com:bigquery-public-data.stackoverflow.post_answers")
+  .load())
+
+# Step (2)
+# Save the `bigqueryDF` as nodes with labels `Person` and `Customer` into Neo4j
+(bigqueryDF.write
+  .format("org.neo4j.spark.DataSource")
+  .mode("ErrorIfExists")
+  .option("url", "neo4j://<host>:<password>")
+  .option("labels", ":Answer")
+  .load())
+----
+=====
+====
+
+=== From Neo4j to BigQuery
+
+[.tabbed-example]
+====
+[.include-with-scala]
+=====
+[source,scala]
+----
+// Step (1)
+// Load `:Answer` nodes as DataFrame
+val neo4jDF: DataFrame = spark.read.format("org.neo4j.spark.DataSource")
+  .option("url", "neo4j://<host>:<password>")
+  .option("labels", ":Answer")
+  .load()
+
+// Step (2)
+// Save the `neo4jDF` as table CUSTOMER into BigQuery
+neo4jDF.write
+  .format("bigquery")
+  .mode("overwrite")
+  .option("temporaryGcsBucket", "<my-bigquery-temp>")
+  .option("table", "<my-project-id>:<my-private-database>.stackoverflow.answers")
+  .save()
+----
+=====
+[.include-with-python]
+=====
+[source,python]
+----
+# Step (1)
+# Load `:Answer` nodes as DataFrame
+neo4jDF = (spark.read.format("org.neo4j.spark.DataSource")
+  .option("url", "neo4j://<host>:<password>")
+  .option("labels", ":Answer")
+  .load())
+
+# Step (2)
+# Save the `neo4jDF` as table CUSTOMER into BigQuery
+(neo4jDF.write
+  .format("bigquery")
+  .mode("overwrite")
+  .option("temporaryGcsBucket", "<my-bigquery-temp>")
+  .option("table", "<my-private-database>.stackoverflow.answers")
+  .save())
+----
+=====
+====
+
+== Exchange Data between RedShift and Neo4j
+
+Amazon Redshift uses SQL to analyze structured and semi-structured data across data warehouses, operational databases,
+and data lakes, using AWS-designed hardware and machine learning to deliver the best price performance at any scale.
+
+=== Pre-requirements
+
+An Amazon RedShift instance up-and-running
+
+==== Dependencies
+
+If you're into the Databricks runtime env you don't need to add any external dependency, otherwise you may need:
+
+* `com.amazon.redshift:redshift-jdbc42:<version>`
+* `org.apache.spark:spark-avro_<scala_version>:<version>`
+* `io.github.spark-redshift-community:spark-redshift_<scala_version>:<version>`
+* `com.amazonaws:aws-java-sdk:<version>`
+
+=== From RedShift to Neo4j
+
+==== In Databricks Runtime
+
+A good starting point in this case is this link:https://docs.databricks.com/data/data-sources/aws/amazon-redshift.html#parameters[Databricks Guide]
```

Comment:
I would like to double check one moment. Will the links to the external resources in this chapter open in a new tab? As a team we try to set general rules regarding links, and we think it would be better for a user if a link to the external resource opens in a new tab.